config/wikis/fawiki.yaml

```diff
@@ -11,6 +11,10 @@ samples:
     2.20k_2015:
         quarry_url: "http://quarry.wmflabs.org/run/59580/output/0/json-lines?download=true"
 
+review_samples:
+    5k_2015:
+        bigger_sample: 20k_2015
```

Comment:
Doesn't specify balanced or unbalanced

Reply:
Yes, if you remember these cases were checked in into wikilabels and labeled autoamtically through the system, so we basically don't have that autolabeling unbalanced part.

Reply:
As discussed on the call today, it seems like this is deeply implicit.  "Review samples" doesn't say anything about merging or autolabeling.  I wonder if we can instead borrow from my work on the modular makefile and instead implement an explicit block to create the merged samples?